### ISSUE TITLE
Document each platforms limits for playing simultaneous videos

### ISSUE THREAD

**Issue by tomprogers** [TheWidlarzGroup/react-native-video#306]
This isn't a complaint -- I'm looking for confirmation of the behavior, and suggesting the relevant details be documented for other devs.

I'm rendering a bunch of looping thumbnail videos to the screen at once, in an endlessly-scrolling grid three wide. Right now I'm attempting to draw 30 of them (i.e. 10 rows). All 30 videos are stored locally on the device, and each is about 90k in size. Only the first 16 play, the others render as transparent and never start playing. The remaining 14 videos all throw errors like this:

```
{ target: 395, error: { code: -11800, domain: 'AVFoundationErrorDomain' } }
```

If there is an underlying limitation that prevents react-native-video from successfully playing more than 16 videos simultaneously, that limit should be documented; if the limit is different on Android (or varies by hardware or OS version or something else), that should also be documented.

Also, I'm very curious about the details of the limitation, but that's beside the point.
## 

FYI: I intend to work around the limitation by using a `<ListView>` and switching playback on/off on a per-row basis as indicated by `onChangeVisibleRows`.

**Comment by mattapperson** [comment]
@tomprogers Can your provide a full stack trace for the errors thrown? There are a couple possible causes for this and I would like to try and narrow it down. 
If you can, even providing a quick sample app to demo the issue and I can track it down from there myself

**Comment by arilitan** [comment]
@tomprogers I have experienced this same issue - its not just playing videos simultaneously it seems that iOS can't handle having 16+ AVPlayer instances in memory at the same time. I don't believe this is anything related to react native, as I had experienced it with a Swift only app. Haven't done too much digging into why its occurring. I worked around the limitation as you mentioned using the onChangeVisibleRows callback. Keep in mind onChangeVisibleRows currently does work for android, i had to fork the listview component to get it to work on android
